scripts/gather-mocked-data.js

```diff
@@ -98,24 +104,47 @@ function writeData(data) {
   });
 }
 
-function gatherMockedData(client, serverAddr) {
+function gatherMigData(client, serverAddr) {
   const namespace = MIG_NAMESPACE;
-  const all = MIG_GROUPS.map(v => {
-    let url = `apis/${v.group}/${v.version}/namespaces/${namespace}/${v.kind}`;
-    let fullUrl = `${serverAddr}/${url}`;
+  const namespaced = MIG_GROUPS.map(v => {
+    const url = `apis/${v.group}/${v.version}/namespaces/${namespace}/${v.kind}`;
     return client
-      .get(fullUrl)
+      .get(url)
       .then(response => {
         return { ...v, data: response.data, url: url, serverAddr: serverAddr };
       })
-      .catch(error => {
-        console.log(error);
+      .catch(err => {
+        console.error(err);
         process.exit(1);
       });
   });
+
   //  Reducing to a single promise that has collected all of the results.
   //  Transform from array to a dict that matches the layout of how we want to store the data
-  return all.reduce((promiseChain, currentTask) => {
+  return namespaced.reduce((promiseChain, currentTask) => {
```

Comment:
This ugly nesting should be cleaned up with async/await, but I think it's fine to merge this as is and file an issue to improve stuff.